### PR TITLE
Qr code component

### PR DESCRIPTION
**Pull request by amos-kibet**
Rework of the challenge.
Nothing changed, except for the commit messages.

### CODE REVIEW

**Review by almirsarajcic — CHANGES_REQUESTED** [review]
Some image is missing or the path is incorrect.
Take a look at the tests.

**Review by almirsarajcic — CHANGES_REQUESTED** [review]
Bring back the `screenshots/.gitkeep` file.
Tests are failing because it's missing.

**Comment by amos-kibet** [comment]
I have removed screenshots folder from .gitignore. 

It was ignored by default, not by me.

**Comment by amos-kibet** [comment]
New learning: I now understand the reason why I always see empty folders having .gitkeep files. git docs say that it only tracks files, so empty folders won't be tracked. Putting an empty .gitkeep file helps to make the folder be tracked by git.

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,3 @@
 # macOS artifacts
 .DS_Store
 
-# Screenshots made during the tests
-/screenshots/**/*
-!/screenshots/.gitkeep
```
Comment (almirsarajcic) [inline .gitignore]: You're not supposed to change `.gitignore` file.

**Review by almirsarajcic — CHANGES_REQUESTED** [review]
You shouldn't commit files in screenshots directory.
Check the diff - it should reflect only the changes in challenges/qr_code_component directory.

File: .gitignore (hunk 1 above)
Comment (amos-kibet) [reply .gitignore]: Okay, I missed that. Let me revert.

Comment (amos-kibet) [reply .gitignore]: Should I keep `!/screenshots/.gitkeep` line in .gitignore?

**Comment by amos-kibet** [comment]
@almirsarajcic I think I have fixed it. Kindly check.

**Review by almirsarajcic — CHANGES_REQUESTED** [review]
Now there are even more changes in the diff.

**Pull request closed without merging**